wiki/models/article.py

```diff
@@ -258,8 +270,14 @@ class Meta:
 
 
 class ArticleRevision(BaseRevisionMixin, models.Model):
-    """This is where main revision data is stored. To make it easier to
-    copy, do NEVER create m2m relationships."""
+    """
+    This is where main revision data is stored. To make it easier to
+    copy, do NEVER create m2m relationships.
+
+    .. pii: This model store IP the addresses of users who have edited the article
```

Comment:
Unless I'm completely misunderstanding the technical language, I think this suggestion would more clearly state what engineers shouldn't do and what the model does. 

```suggestion
    copy, do NOT create m2m relationships.

    .. pii: This model stores the IP addresses of users who have edited the article
```

Reply:
Ha, just copypasta from whenever this was originally written. I'll clean it up, thanks!